README.md

```diff
@@ -0,0 +1,32 @@
+# osu!
+Keep in mind that the package is **not** official, there may be bugs specific to the flatpak. If you find such, report them here.
+> Note: this is lazer, not osu-stable
+
+## Where is the game directory located?
+Check `~/.var/app/sh.ppy.osu/data/osu`
+
+## Graphic tablets
+Currently, support for graphics tablets on linux is implemented through the standalone [OpenTabletDriver](https://github.com/OpenTabletDriver/OpenTabletDriver) driver. \
```

Comment:
it's not standalone though? it's integrated in the game.

Reply:
so right there it was written: `through the standalone driver`

Reply:
Or should I change it to `implemented through standalone driver in game`?

Reply:
OTD isn't a driver, even though the project is incorrectly titled as such.

Something like "Graphics tablet support uses [OpenTabletDriver](...)" should suffice.